toonz/sources/include/tpixel.h

```diff
@@ -154,6 +154,8 @@ class DVAPI DV_ALIGNED(8) TPixelRGBM64
 	Channel b, g, r, m;
 #elif TNZ_MACHINE_CHANNEL_ORDER_MRGB
 	Channel m, r, g, b;
+#elif TNZ_MACHINE_CHANNEL_ORDER_RGBM
+	Channel r, g, b, m;
 #else
```

Comment:
Its needed to compile, Linux could use `TNZ_MACHINE_CHANNEL_ORDER_MRGB` too. But rather focus on fixing build errors first.